tensorbay/opendataset/HalpeFullBody/catalog.json

```diff
@@ -0,0 +1,297 @@
+{
+    "KEYPOINTS2D": {
+        "keypoints": [
+            {
```

Comment:
the `visible` field is not default to `BINARY`,
No `visible` in catalog means keypoint has no `v` field.